packages/aws-fargate/images/inspector/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM node:18-alpine
+ARG NODEJS_VERSION
```

Comment:
qs: From where this value is set?

Reply:
https://github.com/instana/nodejs/pull/1152/files/15ed7d909f067d2e6e6cb507d3fd7f9ca256b9e5#diff-02f198aef71805a77d9836533174d3e061195b4560547217b8fcad4978a8bbbb